### ISSUE TITLE
Zangarmarsh - Twin Spire pvp objective

### ISSUE THREAD

**Issue by Chulain** [ccshiro/cc-buglist#1012]
http://wowwiki.wikia.com/Twin_Spire_Blessing?oldid=1771761
February 11, 2009
Originally, you could not mount with the flag, now you can you just can't fly. 

the flying was added later, you can only carry the flag on a ground mount

**Comment by Chulain** [comment]
2010/12/13 (Patch 4.0.3)
well if it works for something... since the patch 2.4.2 iu can use any mount havin' the flag

posted in 2010 tho

**Comment by Chulain** [comment]
however i think it did work while in flight form

http://wowwiki.wikia.com/Talk:Twin_Spire_Ruins
"I just captured the flag while in flight form off the ground, as well". --Azaram 14:18, 30 March 2008

**Comment by Chulain** [comment]
im guessing flight form was a bug and removed later on and added normal flying mount (you can spam flight form to run away, mounts take time)

imo flag + ground mount is already a bit weird but flying mount is just lame

**Comment by lynxzor** [comment]
Agreed it's pretty lame to be able to fly there with the flag. Also where is my blessing?
![wowscrnshot_070715_160558](https://cloud.githubusercontent.com/assets/795881/8538575/8c53f17e-24c2-11e5-8e36-bb365dd582dd.jpg)

**Comment by Chulain** [comment]
hmm strange i captured the objective on both sides once and i didnt have any problems with the buff

**Comment by lynxzor** [comment]
Also went out of the zone and back in still nada. 

Edit: Joined a party other people in the zone get the buff but I still haven't. 
Edit2: Have died, it's fine now

**Comment by ccshiro** [comment]
Doesn't seem like a bug, in 2.0.3 it had these interrupt flags:
mounting, change_map, immune_or_lost_selection, unk21

In 2.4.3 it has:
change_map, immune_or_lost_selection

Notice how the mount one was removed, as well as unk21 (whatever that might be). Note: unk21 is not related to flying mount, the flag existed on flags in vanilla as well (like 29519).

**Comment by ccshiro** [comment]
Nim and I discussed it, we will readd the flags from 2.0.3.

**Comment by lynxzor** [comment]
http://collab.kpsn.org/display/tc/creature_template
unk21 - Hides nameplates

**Comment by Chulain** [comment]
![wow_2015_08_15_22_01_43_671](https://cloud.githubusercontent.com/assets/10247793/9290665/a711e6cc-439b-11e5-89ff-33fd60e3bc62.jpg)
bugged again

i took the flag while on a flying mount
